docs/user_guide/spin_system/spin_system.rst

```diff
@@ -109,8 +157,9 @@ Consider the example below of a :ref:`coupling_api` object between two sites
         ),
     )
 
-The *site_index* key holds a list of two integers corresponding to the index of the two coupled sites
-within the spin system. The ordering of the integers is irrelevant.
+The *site_index* key holds a list of two integers corresponding to the index of the
+two coupled sites in the ordered list ``sites`` within the SpinSystem object. The
```

Comment:
@pjgrandinetti You often reference ordered/unordered lists, but all list object are ordered in Python. I know you're trying to say the order of elements in a list matters, but this might confuse users. 

There should be a note somewhere which indicates what we mean with ordered vs unordered list

Reply:
is there a set class in Python?

Reply:
There is a set class which does't guarantee order. But still important to get the point about mrsimulator caring about the order of elements in some lists